k8s/db.yaml

```diff
@@ -32,7 +32,8 @@ spec:
     spec:
       containers:
       - name: mysql
-        image: gcr.io/gcb-sandbox/mysql:latest
+        # Dev TODO: add your project id here
+        image: gcr.io/PROJECT-ID/mysql:latest
```

Comment:
(-: this is fastest way to do it without sed or third party tooling...

Reply:
Makes sense to me. These directions look great. 👍